packages/components/src/templates/header.stories.ts

```diff
@@ -36,18 +35,19 @@ export const SampleA = {
         <!-- top-left-area end !-->
         <!-- top-right-area start !-->
         <div class="flex lg:hidden">
-          <sd-navigation-item>
-            <sd-icon name="system/menu" library="global-resources" class="text-xl  -my-[1.5px] -mx-[4px]"></sd-icon>
+          <sd-navigation-item onclick="openDrawerSampleA()">
+            <sd-icon name="system/menu" class="text-xl  -my-[1.5px] -mx-[4px]"></sd-icon>
           </sd-navigation-item>
         </div>
         <!-- top-right-area end !-->
       </div>
       <div class="hidden lg:flex items-end pt-3 justify-between">
         <!-- bottom-left-area start !-->
         <div class="-ml-4">
-          <sd-navigation-item><b>Über Uns</b></sd-navigation-item
-          ><sd-navigation-item><b>Märkte</b></sd-navigation-item
-          ><sd-navigation-item><b>Presseservice</b></sd-navigation-item>
+          <sd-navigation-item class="font-bold">About Us</sd-navigation-item>
+          <sd-navigation-item class="font-bold">Markets</sd-navigation-item>
+          <sd-navigation-item class="font-bold">Sustainability</sd-navigation-item>
+          <sd-navigation-item class="font-bold">Career</sd-navigation-item>
```

Comment:
Please set `href`s here to make this accessible as a link. Apply everywhere.

Reply:
href="javascript:void(0)" has been added to all navigation items